.github/workflows/go.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         run: |
           git config --global user.email you@example.com
           git config --global user.name Your Name
-          make all
-          make test-docker
+          IN_CI=true make all
+          IN_CI=true make test-docker
```

Comment:
Do we have any tests in kpt repo relying on this?

Reply:
There are no tests in the kpt repo that use this; I did some manual testing. For further testing maybe either you could pull this branch to do your own manual testing or I could commit a dummy test just to exercise the new option?